platformio.ini

```diff
@@ -0,0 +1,70 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = main
+
+[env]
+build_flags = 
+    -Iinclude/tls_config
+    -DFRAMEWORK_$PIOFRAMEWORK
+
+[env:esp32dev]
+platform = espressif32
+framework = espidf
+board = esp32dev
+build_flags = 
+    ${env.build_flags}
+    -Iinclude
+    -DMBEDTLS_USER_CONFIG_FILE=\"mbedtls+HKDF.h\"
+board_build.embed_files =
+    components/ena/test/export.bin
+    components/ena-cwa/certs/telekom.pem
+
+
+[env:ID107HR]
+platform = nordicnrf51
+board = ng_beacon ; tbd
+framework = zephyr
+monitor_speed = 115200
+test_ignore = test_desktop
+build_flags = 
+    ${env.build_flags}
+    # For testing: -DUNITY_EXCLUDE_SETJMP_H=1
+    -DEN_INCLUDE_ZEPHYR_DEPS=1
+    -DEN_INIT_MBEDTLS_ENTROPY=0
+lib_deps = exposure-notification
```

Comment:
does not work yet, and probably will be hard to implement. This platform is a 15EUR nRF51822 based over-the-air-flashable fitness tracker bracelet.
Just 32kBytes of RAM and 256kBytes of flash. Might be enough for a "tracking only" device for a few days, with off-loading to and risk calculation on smartphone.